docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -333,11 +278,7 @@ usage: xrdsst token [-h] [-v] {init-keys,list,login} ...
 are enabled, to be able to register authentication certificates and manage Security Server
```

Comment:
Let's remove this part since the user guide is targeted for the Toolkit users and not for the developers.

> X-Road Central Server. In development, this is best run in the mode where auto-approvals
are enabled, to be able to register authentication certificates and manage Security Server
clients without taking separate actions at Central Server, see [UG-CS](#Ref_CS-UG) about

Reply:
Removed, along with UG-CS reference which was no longer used.